keps/sig-node/127-user-namespaces/README.md

```diff
@@ -753,6 +815,17 @@ This section must be completed when targeting beta to a release.
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
+The rollout is just a feature flag on the kubelet and the kube-apiserver.
+
+If one APIserver is upgraded while other's aren't and you are talking to a not upgraded the pod
```

Comment:
I would remove this first sentence - I don't think it matters really and it doesn't bring value.

Reply:
Okay. Sorry, I thought it would help as context, as there are lot of keps that added a field already and we all rely on existing plumbing to handle this correctly. But removed :)